cranelift/codegen/src/ir/function.rs

```diff
@@ -156,10 +168,16 @@ impl Function {
         self.jump_tables.push(data)
     }
 
-    /// Creates a stack slot in the function, to be used by `stack_load`, `stack_store` and
-    /// `stack_addr` instructions.
-    pub fn create_stack_slot(&mut self, data: StackSlotData) -> StackSlot {
-        self.stack_slots.push(data)
+    /// Creates a sized stack slot in the function, to be used by `stack_load`, `stack_store`
+    /// and `stack_addr` instructions.
+    pub fn create_sized_stack_slot(&mut self, data: StackSlotData) -> StackSlot {
```

Comment:
Could you please keep the original name? This is an unnecessary breaking change that would break the nightly cranelift tests of cg_clif.

Reply:
I have no strong opinion, but opposing feedback that I've previously received from @fitzgen (I can't remember in which area) was that we should make it clear it clear to differentiate between fixed and dynamic. If that was just around type naming, then it is a non-issue and I'm happy to revert. It's already been a problem for me while rebasing!

Reply:
I'd prefer we have the explicit name as well; @bjorn3 each Cranelift upgrade is a semver break currently and our APIs are still considered unstable. We shouldn't *needlessly* break compatibility, but getting things right and finding clearer names and less error-prone APIs is a higher priority right now.

Reply:
I guess so @cfallin.